### ISSUE TITLE
Parameterized Consistency Learning-based Deep Polynomial Chaos Neural Network Method for Reliability Analysis in Aerospace Engineering. (arXiv:2203.15655v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#17335]
Parameterized Consistency Learning-based Deep Polynomial Chaos Neural Network Method for Reliability Analysis in Aerospace Engineering. (arXiv:2203.15655v1 [cs.LG])<br>
https://ift.tt/eXZO15q<br>
<p>Polynomial chaos expansion (PCE) is a powerful surrogate model-based reliability analysis method in aerospace engineering. Generally, a PCE model with a higher expansion order is usually required to obtain an accurate surrogate model for some non-linear complex stochastic systems. However, the high-order PCE increases the labeled training data cost for solving the expansion coefficients. To alleviate this problem, this paper proposes a parameterized consistency learning-based deep polynomial chaos neural network (Deep PCNN) method, including the low-order adaptive PCE model (the auxiliary model) and the high-order polynomial chaos neural network (the main model). The expansion coefficients of the high-order main model are parameterized into the learnable weights of the polynomial chaos neural network. The auxiliary model uses a proposed unsupervised consistency loss function to assist in training the main model. The Deep PCNN method can significantly reduce the training data cost in constructing a high-order PCE model without losing surrogate model accuracy by using a small amount of labeled data and many unlabeled data. A numerical example validates the effectiveness of the Deep PCNN method, and the Deep PCNN method is applied to analyze the reliability of two aerospace engineering systems.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
